ui/src/Constant/constant.js

```diff
@@ -1,5 +1,13 @@
 export const FOOTER = {
-    "COPYRIGHT": "Copyright 2022-2023",
-    "RDS_WEBSITE": "RDS Website",
-    "DISCORD_SERVER":"Discord server"
-}
\ No newline at end of file
+  COPYRIGHT: "Copyright 2022-2023",
+  RDS_WEBSITE: "RDS Website",
+  DISCORD_SERVER: "Discord server",
+};
+
+export const NOTIFICATION_TEXT_COLOR = {
+  SUCCESS: "rgb(95, 214, 95)",
+  WARNING: "rgb(238, 241, 41)",
+  ERROR: "rgb(253, 81, 81)",
+};
+
+export const NOTIFICATION_TIMEOUT = 2000;
```

Comment:
Shouldn't this be in the Notification component since these constants are specific it?